packages/components/psammead-radio-schedule/src/testHelpers/helper.jsx

```diff
@@ -11,7 +11,7 @@ export const sentenceCase = text =>
     .charAt(0)
     .toUpperCase() + text.substring(1);
 
-export const stateTypes = ['onDemand', 'live', 'next'];
+export const stateTypes = ['live', 'onDemand', 'onDemand', 'next'];
```

Comment:
Since now we have two `onDemand`s here, in `ProgramCard` test `should render correctly for ${state}` where we go through each state type, we will have two identical snapshots for 'onDemand'. Can you please fix that to avoid repetition?

Reply:
Nice catch, fixed now